src/components/DropDownMenu/DropDownMenu.tsx

```diff
@@ -0,0 +1,49 @@
+import { Person } from '../../types/Person';
+import { DropDownItem } from '../DropDownItem/DropDowItem';
+
+type Props = {
+  people: Person[],
+  hasFocus: boolean,
+  onSelectedPerson: (personSlug: Person) => void,
+  isLoading: boolean
+};
+
+export const DropDownMenu: React.FC<Props> = ({
+  people,
+  hasFocus,
+  onSelectedPerson,
+  isLoading,
+}) => {
+  let content;
+
+  if (!isLoading) {
+    if (people.length) {
+      content = (
+        <div>
+          {people.map(person => (
+            <DropDownItem
+              key={person.slug}
+              person={person}
+              onSelectedPerson={onSelectedPerson}
+            />
+          ))}
+        </div>
+      );
+    } else {
+      content = <p>No matching suggestions</p>;
+    }
+  } else {
+    content = <p>Loading</p>;
```

Comment:
Nice feature, but you are doing debounce a bit wrong
The idea is not to emulate network loading but to reduce the number of filter calls

I would not clear the filter on each new character - probably it is okay to keep old list, until new filtered data is awailable